test/helm/flex_test.go

```diff
@@ -0,0 +1,59 @@
+package helm
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	akov2 "github.com/mongodb/mongodb-atlas-kubernetes/v2/api/v1"
+	"github.com/mongodb/mongodb-atlas-kubernetes/v2/api/v1/common"
+	"github.com/mongodb/mongodb-atlas-kubernetes/v2/test/helper/cmd"
+	"github.com/mongodb/mongodb-atlas-kubernetes/v2/test/helper/decoder"
+)
+
+func TestFlexSpec(t *testing.T) {
+	output := cmd.RunCommand(t, "helm", "template", "--values=flex_values.yaml", "../../helm-charts/charts/atlas-deployment")
```

Comment:
nit: Do we want to check for non-0 exit codes and error out early?

Reply:
this will be catched in `cmd.RunCommand` already 👍 we are passing `t` which will `FailNow` if the command returns with a non-0 status.